perfmetrics/scripts/ls_metrics/listing_benchmark.py

```diff
@@ -568,7 +570,7 @@ def _export_to_bigquery(test_type, config_id, start_time_build, ls_data):
   if len(argv) < 4:
```

Comment:
Can you check if number of arguments is correct. I checked the previous PR but I don't think we increased it with mandatory spreadsheet id.